rust_src/src/keymap.rs

```diff
@@ -12,7 +12,7 @@ use crate::{
     data::{aref, fset, indirect_function, set},
     eval::{autoload_do_load, unbind_to},
     indent::indent_to,
-    keyboard::lucid_event_type_list_p,
+    keyboard::{self, lucid_event_type_list_p},
```

Comment:
In lists you used a different import style for the module itself. We should pick on style and stick with it. I have no preference

Reply:
I'm sticking with the style that doesn't use `self`, then.